### PR TITLE
Remove refreshable store

### PR DESCRIPTION
**Pull request by stevekinney**
Removes refreshable store, uses `stuff`.

### CODE REVIEW

File: src/lib/stores/refreshable.ts [hunk 1]
```diff
@@ -1,37 +0,0 @@
...
-    return cleanUp;
-  };
-
-  const store = writable<PromiseLike<T>>(Promise.resolve(initialData), setUp);
-
-  return {
-    subscribe: store.subscribe,
-    refresh,
-  };
-};
```
Comment (GiantRobots) [inline src/lib/stores/refreshable.ts:37]: 🎉

File: src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/history/compact/__layout.svelte [hunk 2]
```diff
@@ -1,15 +1,28 @@
+<script lang="ts" context="module">
+  import type { LoadInput } from '@sveltejs/kit';
+
+  export const load = async ({ stuff }: LoadInput) => {
+    const { events } = stuff;
+
+    console.log({ events });
```
Comment (GiantRobots) [inline src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/history/compact/__layout.svelte:7]: Extra console.log

File: src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/history/full/event-[id].svelte [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
+    const { attributes } = event;
 
-  const findEvent = async (
-    data: EventualHistoryEvents,
-    id: string,
-  ): Promise<HistoryEventWithId> => {
-    return data.then((events) => events.find((event) => event.id === id));
+    return {
+      props: { attributes },
+    };
```
Comment (GiantRobots) [inline src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/history/full/event-[id].svelte:16]: Same concern as above on not finding an event with the id

File: src/routes/namespaces/[namespace]/workflows/index.svelte [hunk 4]
```diff
@@ -14,53 +17,45 @@
...
-  $: data = refreshable(
-    () => fetchAllWorkflows(namespace, parameters),
-    initialData,
-  );
 </script>
 
 <h2 class="text-2xl">Workflows</h2>
 <WorkflowFilters bind:timeFormat />
-{#await $data}
+{#await workflows}
```
Comment (GiantRobots) [inline src/routes/namespaces/[namespace]/workflows/index.svelte:58]: Dropping refreshable makes a lot of this feel better, glad to see it work out so well

**Review by GiantRobots — APPROVED** [review]

**Comment by stevekinney** [comment]
Closes #268.

**Review by GiantRobots — APPROVED** [review]

**Pull request merged**